sdk/lib/conutils/pager.h

```diff
@@ -26,27 +26,40 @@
 extern "C" {
 #endif
 
-
 // #include <wincon.h>
 
+struct _CON_PAGER;
+typedef BOOL (__stdcall *CON_PAGER_LINE_FN)(
+    IN OUT struct _CON_PAGER *Pager,
+    IN PCTCH line,
+    IN DWORD cch,
+    IN OUT PDWORD pdwFlags);
```

Comment:
This has been modified in accordance too.

Reply:
OK.